src/server/graphql/types/Invoice.js

```diff
@@ -0,0 +1,104 @@
+import {GraphQLFloat, GraphQLID, GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql';
+import {GraphQLEmailType, GraphQLURLType} from 'server/graphql/types';
+import CreditCard from 'server/graphql/types/CreditCard';
+import GraphQLISO8601Type from 'server/graphql/types/GraphQLISO8601Type';
+import InvoiceChargeNextMonth from 'server/graphql/types/InvoiceChargeNextMonth';
+import InvoiceLineItem from 'server/graphql/types/InvoiceLineItem';
+import InvoiceStatusEnum from 'server/graphql/types/InvoiceStatusEnum';
+import connectionDefinitions from 'server/graphql/connectionDefinitions';
+
+
+/* Each invoice has 3 levels.
+ * L1 is a the invoice itself: how much to pay.
+ * L2 is line items (next month charges, added users, removed users, inactivity credits, previousBalance) with a quantity
+ * L3 is a detailed line item & is a breakdown of the L2 quantity (eg a user with the pause/unpause dates)
+ */
+
+const Invoice = new GraphQLObjectType({
+  name: 'Invoice',
+  description: 'A monthly billing invoice for an organization',
+  fields: () => ({
```

Comment:
i'm foggy on the rules here, i _think_ i don't use it as much because if i change something like `orgName` then it means i have to return `total` as well, but now that i think of it, I believe that was a limitation in using the changefeeds since the subscribed fields were static (i had to request all of them). let's experiment with using it more in a few cases & see if it works!